lib/barbeque/message/job_execution.rb

```diff
@@ -7,6 +7,19 @@ class JobExecution < Base
       attr_reader :application # [String] To specify `job_definitions.application_id`
       attr_reader :job         # [String] To specify `job_definitions.name`
 
+      # Creates JobExecution messages from a notification which came from subscribed SNS topic
+      def self.create_message_from_sns_notification(raw_message, message_body, job_queue:)
```

Comment:
Okay, I intorduce Notification message handler and message.
How about this? 8ab1fbe1ac183b79532decea14f21394149d1428